pyvista/core/filters/poly_data.py

```diff
@@ -2147,7 +2307,7 @@ def extrude(poly_data, vector, inplace=False, progress_bar=False):
         >>> import pyvista
```

Comment:
For future reference, the `extrude*` docstrings could do with a thorough facelift. There are several VTK-only references that don't apply to our interfaces.

Reply:
Fixed this and added a bonus example for ``extrude_rotate``:

```py
import numpy as np
points = np.array([[-0.18, 0, 0],
                   [-0.18, 0, 0.01],
                   [-0.18, 0, 0.02],
                   [-0.01, 0, 0.03],
                   [-0.01, 0, 0.04],
                   [-0.02, 0, 0.5],
                   [-0.05, 0, 0.75],
                   [-0.1, 0, 0.8],
                   [-0.2, 0, 1.0]])
spline = pyvista.Spline(points, 30)
extruded = spline.extrude_rotate(resolution=20)
extruded.plot(color='tan')
```

![image](https://user-images.githubusercontent.com/11981631/125210238-aa2e6280-e25b-11eb-9e2d-ea6b0af31542.png)